src/test/java/org/apache/sling/discovery/oak/its/setup/SimulatedLeaseCollection.java

```diff
@@ -70,6 +70,15 @@ public void incSeqNum(int amount) {
 
     public synchronized void hooked(SimulatedLease lease) {
         leases.add(lease);
+//        if (leases.size() > 1) {
+//            incSeqNum();
+//        }
```

Comment:
Remove?
```suggestion
```

Reply:
yes, should remove - left over from testing